Sprint-3/revise/implement/password-validator.js

```diff
@@ -0,0 +1,24 @@
+function checkPassword(password) {
+    if (password.length < 5) {
+      return false;
+    }
+  
+    if (!/[A-Z]/.test(password)) {
+      return false;
+    }
+  
+    if (!/[a-z]/.test(password)) {
+      return false;
+    }
+  
+    if (!/\d/.test(password)) {
+      return false;
+    }
+  
+    if (!/[!#$%.*&]/.test(password)) {
+      return false;
+    }
+  
```

Comment:
Looks good. How do you plan to check if the password is one of the passwords already in used?

You can assume these "used passwords" are stored in some array.